services/copyright-service/.env

```
@@ -0,0 +1,51 @@
+API_GATEWAY_RETRIES=3
+GATEWAY_HEALTH_PATH=/actuator/health
+GATEWAY_CONTEXT_PATH=/api/v1
+
+# Services
+IDENTITY_SERVICE_URL=http://localhost:8081
+ANALYTICS_SERVICE_URL=http://localhost:8082
+NOTIFICATION_SERVICE_URL=http://localhost:8083
+
+# Security
+JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
+JWT_EXPIRES_IN=24h
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES_IN key should go before the JWT_SECRET key

(UnorderedKey)